test/test_heuristics.rb

```diff
@@ -7,9 +7,13 @@ def fixture(name)
     File.read(File.join(samples_path, name))
   end
 
-  def file_blob(name)
+  def file_blob(name, alt_name=nil)
     path = File.exist?(name) ? name : File.join(samples_path, name)
-    FileBlob.new(path)
+    blob = FileBlob.new(path)
+    if !alt_name.nil?
+      blob.instance_variable_set("@path", alt_name)
+    end
+    blob
```

Comment:
Could you comment this change? I'm not sure I understand why it's required.

Reply:
This was intended to test samples with alternative extensions. In this case, testing that any JavaScript file, if it had a `.gs` extension (as in Google Script stuff), wouldn't be classified as GLSL.

Alternatively, we could just add more actual JavaScript with `.gs` extension as found in the wild.